fastlane_core/lib/fastlane_core/configuration/configuration_file.rb

```diff
@@ -78,5 +79,48 @@ def method_missing(method_sym, *arguments, &block)
         end
       end
     end
+
+    # Override configuration for a specific lane.
+    #
+    # lane_name  - Symbol representing a lane name.
+    # block - Block to execute to override configuration values.
+    #
+    # Discussion If received lane name does not match the lane name available as environment variable, no changes will
```

Comment:
Hm, this was from the original PR. I can make the comments look more like our usual documentation. 👍